contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -1366,6 +1366,15 @@ exec_stmt_decl_table(PLtsql_execstate *estate, PLtsql_stmt_decl_table *stmt)
 		}
 
 		tblname = psprintf("%s_%d", var->refname, estate->nestlevel);
+
+		if (strlen(tblname) > 63)
+		{
+			// truncate tblname to fit the "_#" nestlevel suffix
+			tblname[63-(strlen(tblname)-63)] = '\0';
```

Comment:
instead of using 63, can express it as NAMEDATALEN (which is 64)?

also an edge case, but if estate->nestlevel is double digits, we will still be over 64 chars

Reply:
I'll fix this to use NAMEDATALEN.

As for the edge case, this already accounts for the possibility of having a larger nest level - I am just subtracting the difference in length between the refname and tblname, which is exactly how long the nestlevel suffix is, whether it's one, two, or more digits.